.circleci/config.yml

```diff
@@ -420,6 +420,26 @@ jobs:
           root: *working_directory
           paths:
             - packages/multi-format/coverage/
+  test-payment-detection:
```

Comment:
you need to add this as a dependency for the publish and release jobs, all the way at the bottom of the file.